openvdb_houdini/houdini/Utils.h

```diff
@@ -305,10 +305,45 @@ bool isLogForwarding(OP_OpTypeId);
 /// @}
 
 
+////////////////////////////////////////
+
+
+// Grid type lists, for use with openvdb::GridBase::apply()
+
+using ScalarGridTypes = openvdb::TypeList<
+    openvdb::BoolGrid,
```

Comment:
No MaskGrid?

Reply:
It's not one of the supported Houdini grid types.

Reply:
Ah, I missed that this is in houdini/Utils.h. I've actually had a request to use a method like this outside of Houdini, so I think these import statements should live in the core library and be non-Houdini specific. 

Perhaps they could be defined in an `openvdb::` namespace based on the grid types being registered and then redefined in an `openvdb_houdini::` namespace but with any grids Houdini doesn't support pulled out?

```
namespace openvdb_houdini {
using ScalarGridTypes = openvdb::ScalarGridTypes::Remove<MaskGrid>;
using AllGridTypes = openvdb::AllGridTypes::Remove<MaskGrid, StringGrid>;
}
```

Reply:
I intentionally left the type lists out of the core library so as not to bless certain grid types (any more than we already have) and give the impression that core library code, at least, need not support other types or other tree configurations.